src/dora/dora_launcher.star

```diff
@@ -123,7 +123,7 @@ def get_config(
     if dora_params.image != "":
         IMAGE_NAME = dora_params.image
     else:
-        IMAGE_NAME = "ethpandaops/dora:latest"
+        IMAGE_NAME = "ethpandaops/dora:master"
```

Comment:
I'd prefer leaving this as `ethpandaops/dora:latest` by default.
That's the stable image that should be used for anyone not working on devnet 4, using kurtosis for research or other deneb related testing. 
They should not be affected by bugs introduced by unstable master builds.

Maybe add a condition, if electra is to be activated, use master,  otherwise use latest?